### ISSUE TITLE
Roundhouse dotnet tool does not run

### ISSUE THREAD

**Issue by Roemer** [cake-build/cake#2465]
### What You Are Seeing?
Using the Roundhouse .Net Core tool with `DotNetCoreTool` does not work as it says `No executable found matching command "dotnet-roundhouse`. This is probably as the Roundhouse .Net Core tool is installed as `rh` and the alias that is created in the cake folder is `rh.ex`.

### What is Expected?
The tool should run correctly.

### What version of Cake are you using?
0.32.1

### Are you running on a 32 or 64 bit system?
64

### What environment are you running on?  Windows? Linux? Mac?
Windows

### Are you running on a CI Server?  If so, which one?
No

### How Did You Get This To Happen? (Steps to Reproduce)
1. Create a new cake project.
2. Add `#module nuget:?package=Cake.DotNetTool.Module`
3. Run `build.ps1 --bootstrap`
4. Add `#tool dotnet:?package=dotnet-roundhouse`
5. Run `DotNetCoreTool("roundhouse");` from any cake task.

**Comment by gep13** [comment]
@Roemer I have just created a PR that should address this for you.

**Comment by gep13** [comment]
@Roemer hmm, I had made an assumption based on the name of the package.  Having just attempted the installation of this global tool, I can see that this PR isn't going to work as it currently stands.

**Comment by gep13** [comment]
@Roemer actually, this should already work for you....

Try the following:

![image](https://user-images.githubusercontent.com/1271146/52169152-f2acd200-272b-11e9-953d-9f8e6fd840c8.png)

There should be no need to run `DotNetCoreTool("roundhouse");`

**Comment by gep13** [comment]
@Roemer I have updated the PR to include only the shims that are created as part of the dotnet tool installation, i.e. `rh.exe` and `rh`.

As I mentioned though, for you, on Windows, this should already work today:

![image](https://user-images.githubusercontent.com/1271146/52169200-81b9ea00-272c-11e9-8fb0-80d663039375.png)

**Comment by Roemer** [comment]
@gep13 Hmm yes it works now. I think I first tried to use the nuget package (`tool nuget:?package=roundhouse`) which did not work for  me because there was a bug in the executable so I tried using the .net core tool (which worked). For some reason I had to add `ToolPath = Context.Tools.Resolve("rh.exe")` to `RoundhouseSettings` but now it seems to work correctly. Probably because I now have deinstalled the nuget tool and only installed the .Net Core tool.
